includes/shortcodes.php

```diff
@@ -892,6 +896,18 @@ function give_form_grid_shortcode( $atts ) {
 		$form_args['tax_query'][] = $tax_query;
 	}
 
+	// Maybe filter by form Amount Donated.
+	if( !empty( $atts['orderby'] ) && 'amount_donated' === $atts['orderby'] ) {
```

Comment:
@kakshak We do not need to use multiple `if` conditions of `orderby`. Use switch case instad

Reply:
@ravinderk I've updated the condition with `orderby` instead of `if`